.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.10.cjs"
```

Comment:
dq: why is this needed? 

I thought this PR would get rid of yarn `1.x` ^^

Reply:
Nice catch!!! It actually does, but as I was updating the pathing didnt get upgraded properly! Probably because of the order I did `yarn set version <version>` in.

Reply:
Fixed! Thanks for the catch.